### PR TITLE
Correctly provide swagger UI as a build on Circle

### PR DESCRIPTION
**Pull request by Cadair**
This clones the swagger UI and then outputs a redirected URL so swagger works at the end :)

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -4,12 +4,25 @@ gendoc: &gendoc
...
+  name: Build Swagger UI
+  command: |
+    ls scripts/
+    mkdir -p api/client-server
+    git clone https://github.com/matrix-org/swagger-ui swagger-ui
+    cp -r swagger-ui/dist/* api/client-server/
+    mkdir -p api/client-server/json
+    cp scripts/swagger/api-docs.json api/client-server/json/
+    wget https://raw.githubusercontent.com/matrix-org/matrix.org/master/content/swagger.css -O api/client-server/swagger.css
+    patch api/client-server/index.html scripts/swagger-ui.patch
```
[the remark this answers is not in the document]
Comment (Cadair) [reply .circleci/config.yml]: I had to change it. Could probably merge the changes into the matrix.org repo though.

[the remark this answers is not in the document]
Comment (turt2live) [reply .circleci/config.yml]: +1 to that :)

File: .circleci/config.yml [hunk 2]
```diff
@@ -4,12 +4,26 @@ gendoc: &gendoc
...
+  name: Build Swagger UI
+  command: |
+    ls scripts/
+    mkdir -p api/client-server
+    git clone https://github.com/matrix-org/swagger-ui swagger-ui
+    cp -r swagger-ui/dist/* api/client-server/
+    mkdir -p api/client-server/json
+    cp scripts/swagger/api-docs.json api/client-server/json/
+    wget https://raw.githubusercontent.com/matrix-org/matrix.org/master/content/swagger.css -O api/client-server/swagger.css
+    wget https://raw.githubusercontent.com/matrix-org/matrix.org/master/scripts/swagger-ui.patch
```
Comment (turt2live) [inline .circleci/config.yml]: I think you'll need to `-O scripts/swagger-ui.patch` on this

Comment (Cadair) [reply .circleci/config.yml]: I changed the line below to read the patch file from the cwd.

**Pull request merged**